editor/project_manager.cpp

```diff
@@ -996,8 +1004,8 @@ void ProjectManager::_unhandled_input(const Ref<InputEvent> &p_ev) {
 		switch (k->get_scancode()) {
 
 			case KEY_ENTER: {
-
-				_open_project();
+				if (!npdialog->get_popup_window_active())
```

Comment:
Wouldn't something like `!npdialog->is_visible_in_tree()` be sufficient, and avoid the need to introduce a new `set_popup_window_active` API just for `ProjectDialog`?

Otherwise, we could look into exposing `Popup`'s private `popped_up` variable, but if visibility is equivalent, that should be fine with `CanvasItem::is_visible_in_tree()`.